Prefabs/Player/Tanks/Tank.tscn

```diff
@@ -56,10 +56,12 @@ __meta__ = {
 }
 
 [node name="AnimatedSprite" type="AnimatedSprite" parent="." groups=["player"]]
+z_index = 5
```

Comment:
I think it would be enough to put this property in the parent, in `KinematicBody2d`, and it would have the same effect